src/style.css

```diff
@@ -0,0 +1,93 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
```

Comment:
You have declared these fonts but you never use them.
You can set for `body` for example `font-family`. So then your `inherit` rule will work.

Reply:
I've tried it now, it still doesn't work.
I used font-family: "Avenir", sans-serif;

Reply:
So, I see you have added `font-family` to the body. And now it looks well